packages/editor/src/lib/primitives/Vec.ts

```diff
@@ -531,6 +531,9 @@ export class Vec {
 	static Average(arr: VecLike[]) {
 		const len = arr.length
 		const avg = new Vec(0, 0)
+		if (len === 0) {
```

Comment:
@MitjaBezensek can you add a test for this in `Vec.test.ts`? :)

cc @SomeHats because we ran into a similar issue on this with the `Uni` function if you recall